test/transforms.jl

```diff
@@ -1,6 +1,45 @@
 @testset "Transforms" begin
   # using MersenneTwister for compatibility between Julia versions
   rng = MersenneTwister(42)
+  @testset "Levels" begin
+    a = categorical(["yes","no","no","no","yes"]) 
+    b = categorical([1,2,4,2,8],ordered=false) 
+    c =categorical([1,2,1,2,1]) 
+    d= [1,23,5,7,7]
+    t = Table(; a, b, c, d)
+
+    T = Levels(:a => ["yes","no"], :c => [1,2,4], :d => [1,23,5,7])
+    T_same = Levels("a" => ["yes","no"], "c" => [1,2,4], "d" => [1,23,5,7])
+    n,c = apply(T,t)
+    n,c = apply(T_same,t)
+    @test levels(n.a) == ["yes","no"]
+    @test levels(n.c) == [1,2,4]
+    tₒ = revert(T, n, c)
+    @test levels(tₒ.a) == ["no","yes"]
+    @test levels(tₒ.c) == [1,2]
+
+    T = Levels(:a => ["yes","no"], :c => [1,2,4], :d => [1,23,5,7],ordered=[:b])
```

Comment:
Please review all whitespaces in this PR.
```suggestion
    T = Levels(:a => ["yes","no"], :c => [1,2,4], :d => [1,23,5,7], ordered=[:b])
```